### ISSUE TITLE
Destroy on SSH does not cleanup .chef/provisioning/ssh

### ISSUE THREAD

**Issue by jonsmorrow** [opscode-cookbooks/delivery-cluster#98]
Not sure if this is our issue or something we need to fix in the ssh driver but yeah we should have a clean env after a destroy.

**Comment by afiune** [comment]
I agree.. We could address it on the `Rakefile` for now but at the end I think it belongs to the SSH Driver.

**Comment by jonsmorrow** [comment]
Yes lets do that and i'll file a ticket on ssh too and link here.

**Comment by afiune** [comment]
@double-z what about this issue?
